### DISCUSSION TITLE
any fix for hovering over start button? it turns gray not blue

### DISCUSSION THREAD

**Discussion by Rezy774** [valinet/ExplorerPatcher#2343] [Q&A]
NOTHING ELSE IS BROKEN IM ON LATEST EXPLORER PATCHER AND VERSION 22621.2428 OF WINDOWS 11

**Comment by Rezy774** [comment]
https://github.com/valinet/ExplorerPatcher/assets/128003302/55e173fe-96a7-4858-88ef-eb233e59f5d9

**Comment by pyrates999** [comment]
Please post explorer patcher version.  Don't just say you're on the latest version.
